src/ClientData/ProcessDataTest.cpp

```diff
@@ -190,6 +193,75 @@ TEST(ProcessData, FindModuleBuildIdsByPath) {
   EXPECT_TRUE(process.FindModuleBuildIdsByPath("not/loaded/module").empty());
 }
 
+TEST(ProcessData, FindModulesByFilename) {
+  constexpr const char* kFileName1 = "file1";
+  constexpr const char* kFilePath1 = "path/to/file1";
+  constexpr const char* kBuildId1 = "buildid1";
+  constexpr uint64_t kStartAddress1 = 0;
+  constexpr uint64_t kEndAddress1 = 10;
+  ModuleInfo module_info_1;
+  module_info_1.set_file_path(kFilePath1);
+  module_info_1.set_build_id(kBuildId1);
+  module_info_1.set_address_start(kStartAddress1);
+  module_info_1.set_address_end(kEndAddress1);
+
+  constexpr const char* kFileName2 = "file2";
+  constexpr const char* kFilePath2 = "path/to/file2";
+  constexpr const char* kBuildId2 = "buildid2";
+  constexpr uint64_t kStartAddress2 = 100;
+  constexpr uint64_t kEndAddress2 = 110;
+  ModuleInfo module_info_2;
+  module_info_2.set_file_path(kFilePath2);
+  module_info_2.set_build_id(kBuildId2);
+  module_info_2.set_address_start(kStartAddress2);
+  module_info_2.set_address_end(kEndAddress2);
+
+  constexpr const char* kFilePath3 = kFilePath2;
```

Comment:
What about just using `kFilePath2` instead of declaring new constants? Otherwise the connection between `kFileName2` and `kFilePath3` is not clear from the names. Same for `kFilePath4`.

Reply:
Sounds good to me.